### PR TITLE
Made the profiler page better

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Comment by Nyholm** [comment]
I was 30 seconds to slow to commit this..

File: src/DataCollector/CacheDataCollector.php [hunk 1]
```diff
@@ -128,7 +128,7 @@ private function calculateStatistics()
                 $statistics[$name]['time'] += $call->time;
                 if ($call->name === 'getItem') {
                     $statistics[$name]['reads'] += 1;
-                    if ($call->result !== false) {
+                    if ($call->isHit) {
```
Comment (cryptiklemur) [inline src/DataCollector/CacheDataCollector.php:131]: should be a method?

File: src/Cache/LoggingCachePool.php [hunk 2]
```diff
@@ -155,4 +157,27 @@ public function getCalls()
...
+    {
+        $type = gettype($value);
+        if (in_array($type, ['boolean', 'integer', 'double', 'string', 'NULL'])) {
+            $rep = $value;
+        } elseif ($type === 'array') {
+            $rep = json_encode($value);
+        } elseif ($type === 'object') {
+            $rep = get_class($value);
+        } else {
+            $rep = sprintf('<DATA:%s>', $type);
```
Comment (cryptiklemur) [inline src/Cache/LoggingCachePool.php:178]: maybe we could try running through jms_serializer, if the user has it installed

File: src/Resources/views/Collector/cache.html.twig [hunk 3]
```diff
@@ -91,8 +100,12 @@
                     <li class="{{ i is odd ? 'odd' : 'even' }}">
                         <div>
                             <strong>Name</strong>: {{ call.name }}<br />
-                            <strong>Arguments</strong>: {{ call.arguments|json_encode() }}<br />
-                            <strong>Results</strong>: {{ call.result|json_encode() }}<br />
+                            <strong>Arguments</strong>: {{ call.arguments|json_encode|truncate(140) }}<br />
+                            <strong>Results</strong>: {% if call.result == false %}
+                                <span class="color-red">Miss</span>
+                            {% else %}
+                                {{ call.result|truncate(140) }}
```
Comment (cryptiklemur) [inline src/Resources/views/Collector/cache.html.twig:107]: may make sense to have a button to click to show the rest at some point too

File: src/DataCollector/CacheDataCollector.php (hunk 1 above)
Comment (Nyholm) [inline src/DataCollector/CacheDataCollector.php:131]: No, It is a property that only exists if call->name == 'getItem*

File: src/Cache/LoggingCachePool.php (hunk 2 above)
Comment (Nyholm) [inline src/Cache/LoggingCachePool.php:178]: You are talking about if $type=='object'? Yeah, why not.

On line 178 we do only have 'resource' and some other thing we dont want to display.

File: src/Resources/views/Collector/cache.html.twig (hunk 3 above)
Comment (Nyholm) [inline src/Resources/views/Collector/cache.html.twig:107]: Yes. Agree. But I left that for a future improvement.

File: src/Cache/LoggingCachePool.php (hunk 2 above)
Comment (cryptiklemur) [inline src/Cache/LoggingCachePool.php:178]: yeah

**Pull request merged**